ivy/functional/frontends/torch/nn/functional/vision_functions.py

```diff
@@ -120,3 +122,78 @@ def pad(input, pad, mode="constant", value=0):
                 + "['constant', 'reflect', 'replicate', 'circular']"
             ).format(mode)
         )
+
+@to_ivy_arrays_and_back
+def upsample_bilinear(input, size=None, scale_factor=None):
```

Comment:
Function works unexpectedly when upsampling input to size = (n,m) where n is not equal to m.